index.js

```diff
@@ -404,15 +404,19 @@ export default function(value, nominatim_object, optional_conf_parm) {
      */
     function getRegexKeyForKeyFromOsmDefaults(key) {
         var regex_key;
+        var exact_match = false;
 
-        for (var osm_key in osm_tag_defaults) {
+        Object.keys(osm_tag_defaults).forEach(function (osm_key) {
+            if (exact_match === true) {
+                return;
+            }
             if (key === osm_key) { // Exact match.
                 regex_key = osm_key;
-                break;
```

Comment:
Yes but the behavior is inconsistent between browsers... especialy old browsers... like IE11 for example ^^

a return in a `forEach` stop the loop only on some modern browsers like chrome or firefox.

<img width="550" alt="capture d ecran 2017-03-21 a 09 46 21" src="https://cloud.githubusercontent.com/assets/54712/24139202/7e6041b4-0e1b-11e7-8a2d-8ba523d899de.png">